control/server.py

```diff
@@ -204,6 +226,39 @@ def _start_spdk(self):
         for trtype in spdk_transports.split():
             self._create_transport(trtype.lower())
 
+    def _stop_spdk(self):
+        """Stops SPDK process."""
+        assert self.spdk_process is not None # should be verified by the caller
+
+        return_code = self.spdk_process.returncode
+        rpc_socket = self.config.get("spdk", "rpc_socket")
+
+        # Terminate spdk process
+        signal.signal(signal.SIGCHLD, signal.SIG_DFL)
+        if return_code is not None:
+            self.logger.error(f"SPDK({rpc_socket}) pid {self.spdk_process.pid} already terminated, exit code: {return_code}")
```

Comment:
nit: why do we need `rpc_socket` in this log message (and below)? I would rather have it like `"SPDK({pid}) ..."`

Reply:
@trociny 🖖 rpc socket is logged just for a narrow case of multi_gateway test, to differentiate between gatewayA and gatewayB. I would not mind promoting [gateway_name](https://github.com/baum/ceph-nvmeof/blob/21b77ce84ad6364d26a3f671b80189b13b220c51/control/server.py#L87) to be a self member and print it instead. WDYT?

Reply:
The gateway name looks more useful for me than the rpc socket, which I expect usually to be the same in production. Maybe we could add the gateway name as a log prefix?